addons/sys_intercom/XEH_postInitClient.sqf

```diff
@@ -8,6 +8,9 @@ if (!isClass (configFile >> "CfgPatches" >> "ace_interact_menu")) exitWith {};
 ["Tank", "init", FUNC(infantryPhoneAction), nil, nil, true] call CBA_fnc_addClassEventHandler;
 ["Car_F", "init", FUNC(infantryPhoneAction), nil, nil, true] call CBA_fnc_addClassEventHandler;
 
+["Tank", "init", FUNC(passengerIntercomAction), nil, nil, true] call CBA_fnc_addClassEventHandler;
+["Car_F", "init", FUNC(passengerIntercomAction), nil, nil, true] call CBA_fnc_addClassEventHandler;
```

Comment:
This function will add it on every single init of every single vehicle. It does not check if it was already added like `infantryPhoneAction` does. I would just merge this into the other function. Or even call each from a common function which makes sure it gets added once, and based on what it is supposed to have calls the add actions functions.